lib/screens/config/dup/override/free_text.ex

```diff
@@ -7,6 +7,7 @@ defmodule Screens.Config.Dup.Override.FreeText do
           | %{route: route_pill}
           | %{color: color, text: String.t()}
           | %{special: special}
+          | %{icon: Screens.Config.Dup.Override.FreeTextLine.icon()}
```

Comment:
Would it make sense to remove the B/C/D/E options for the `route_pill` type below now, since we have the circular icon versions instead? Same thing for the frontend `RoutePill` component.

Reply:
We sometimes use GL·B pills, so maybe it should be that instead? I can change the text and make the font size smaller for those.